bundles/org.openhab.binding.tr064/README.md

```diff
@@ -1,14 +1,16 @@
 # TR-064 Binding
 
-This binding brings support for internet gateway devices that support the TR-064 protocol.
+This binding brings support for internet gateway devices that support the TR-064 protocol. In particular, it supports
+the AVM FritzBox family of routers. 
 It can be used to gather information from the device and/or re-configure it.
+Even though textual configuration is possible, it is strongly recommended to use the Main User Interface for configuration.
 
 ## Supported Things
 
 Four thing types are supported:
 
 - `generic`: the internet gateway device itself (generic device)
-- `fritzbox`: similar to `generic` with extensions for AVM FritzBox devices
+- `fritzbox`: similar to `generic` with extensions for AVM FritzBox devices. This is also a `rootDevice`
```

Comment:
I would suggest to split that in two parts: "Bridges" and "Things" and remove the `rootDevice` reference at all. It is a leftover from an early development stage. Later the `rootDevice` was split into `generic` and `fritzbox`.

Reply:
I tried to rephrase the paragraph